tests/integration/targets/idrac_secure_boot/tests/attributes.yaml

```diff
@@ -0,0 +1,520 @@
+# Dell OpenManage Ansible modules
+# Copyright (C) 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- name: Test that we have an iDRAC host, iDRAC username and iDRAC password
+  ansible.builtin.fail:
+    msg: 'Please define the following variables: idrac_ip, idrac_user and
+     idrac_password.'
+  when: 'idrac_ip is not defined or idrac_user is not defined or idrac_password
+   is not defined'
+
+- block:
+    - name: Set boot mode to Uefi (check mode - changes expected)
+      dellemc.openmanage.idrac_secure_boot: &boot_md_uefi
+        idrac_ip: "{{ idrac_ip }}"
+        idrac_user: "{{ idrac_user }}"
+        idrac_password: "{{ idrac_password }}"
+        validate_certs: false
+        boot_mode: Uefi
+        restart: true
+        restart_type: GracefulRestart
+      register: cm_changes
+      check_mode: true
+
+    - name: Verify - Set boot mode to Uefi (check mode - changes expected)
+      ansible.builtin.assert:
+        that:
+          - cm_changes.changed
+          - cm_changes.msg == 'Changes found to be applied.'
+
+    - name: Set boot mode to Uefi (normal mode)
+      dellemc.openmanage.idrac_secure_boot:
+        <<: *boot_md_uefi
+      register: boot_md_uefi
+
+    - name: Verify - Set boot mode to Uefi (normal mode)
+      ansible.builtin.assert:
+        that:
+          - boot_md_uefi.changed
+          - boot_md_uefi.msg == 'Successfully updated the iDRAC Secure Boot settings.'
+
+    - name: Get details by uri.
+      ansible.builtin.uri:
+        url: "https://{{ idrac_ip | ansible.utils.ipwrap
+         }}/redfish/v1/Systems/System.Embedded.1/Bios"
+        user: "{{ idrac_user }}"
+        password: "{{ idrac_password }}"
+        validate_certs: false
+        status_code: 200
+      register: boot_md_uri
+
+    - name: Verify -  Get details by uri.
+      ansible.builtin.assert:
+        that:
+          - boot_md_uefi.changed
+          - 'boot_md_uri.json.Attributes.BootMode == "Uefi"'
+
+    - name: Set boot mode to bios (check mode - changes expected)
```

Comment:
Change the sequence of tasks. First set to BIOS and then UEFI

Reply:
done